hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/TestWriteCopyOnWrite.java

```diff
@@ -382,7 +386,8 @@ public void testInsertAsyncClustering() throws Exception {
   @Test
   public void testInsertWithSmallBufferSize() throws Exception {
     // reset the config option
-    conf.setDouble(FlinkOptions.WRITE_TASK_MAX_SIZE, 200 + getBatchSize());
+    conf.set(FlinkOptions.WRITE_MEMORY_SEGMENT_PAGE_SIZE, 64);
```

Comment:
why the `WRITE_MEMORY_SEGMENT_PAGE_SIZE` matters here?

Reply:
It's tuned to match the buffer size setting previously for testing minibatch flush.

Reply:
Can you add some doc for it.

Reply:
ok